### ISSUE TITLE
Publish GDS layer map

### ISSUE THREAD

**Issue by Laksen** [google/skywater-pdk#91]
Currently there doesn't seem to be any way to find the GDS layer id for a given layer.

**Comment by RTimothyEdwards** [comment]
@Laksen :  This document: https://docs.google.com/spreadsheets/d/176OcP2h_TT_pDP2-zBLijlgFUP38LVgMQPlDtpLcrsk/edit?ts=5eff8387#gid=459375686 doesn't tell you what you want to know?

**Comment by smunaut** [comment]
Also see https://github.com/google/skywater-pdk/pull/89 that aims to merge that document inside this repo.

**Comment by Laksen** [comment]
@RTimothyEdwards yes, thank you, that contains the relevant information. And with the PR @smunaut linked it should be easy to generate a layer map from it

**Comment by mithro** [comment]
Adding the resistance and capacitance values in https://github.com/google/skywater-pdk/pull/106

**Comment by RTimothyEdwards** [comment]
@mithro :  Is there anything that actually needs to be done here for this issue?

**Comment by 20Mhz** [comment]
maybe ASCII map files?

**Comment by rovinski** [comment]
@20Mhz there isn't really a standard for ASCII map files, and every tool seems to have a different format for them.

#89 added a CSV file version of the Google doc. IMO if there are any other versions of a GDS layer map, they should be extracted from that CSV via script to maintain a singular layer map source file.
